### ISSUE TITLE
[Screen Reader-Home Page]: 'Skip to Content' link is not defined on MakeCode home page.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2567]
**User Experience:**
Screen reader users who visit several pages on the same site can avoid having to hear all heading graphics and dozens of navigation links on every page before the main content is spoken.

**Test Environment:**
OS: Windows 10 RS2
Build: 15063.2
Browser: Edge, FF

**Steps to reproduce:**
1-Navigate to url:- https://makecode.com/
2-Navigate through all controls (say computing education, about Microsoft makecode, get inspired, resources elements etc)lying on the page.
3-Verify, 'Skip to Content' link is defined or not on Home page

**Actual Behavior:**
'Skip to Content' link is not defined on MakeCode home page. Due to this screen reader user visit several pages on the same site can not avoid to hear all the header information.

**Expected Behavior:**
'Skip to Content' link should be defined on MakeCode home page.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={20bce2aa-9952-4a10-af79-3376bcc8766d}

**Please refer attachment for more details.**
![mas2 4 1_home page_ skip to content link is not defined](https://user-images.githubusercontent.com/30434417/28870884-a58d681e-779f-11e7-91ff-0d6a362ddb90.png)

**Comment by microsoftsam** [comment]
Fixed in makecode.com

**Comment by abchatra** [comment]
@microsoftsam if fixed add the fixed label

**Comment by microsoftsam** [comment]
Will do. Thanks!

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.com/ and issue is fixed now. Hence closing the bug.
